python/tskit/tables.py

```diff
@@ -3451,6 +3451,49 @@ def union(
             )
 
     def find_ibd(self, samples, max_time=None, min_length=None):
+        """
+        Returns a dictionary containing all IBD (identical-by-descent) segments for the
+        supplied list of sample pairs.
+
+        A pair of nodes ``(u, v)`` has an IBD segment with a left and right coordinate
+        ``[left, right)`` and ancestral node ``a`` iff the most recent common ancestor
+        of the segment ``[left, right)`` in nodes ``u`` and ``v`` is ``a``, and the
+        segment has been inherited along the same genealogical path (ie. it has not
+        been broken by recombination). The segments returned are the longest possible
+        ones.
+
+        Note that this definition is purely genealogical -- allelic states
+        *are not* considered here. If used without time or length thresholds, the
+        segments returned for a given pair will partition the span of the contig
+        represented by the tree sequence.
+
+        Each item in the outputted dictionary object contains the IBD segments for a
+        particular sample pair. Suppose the output is
+        ```{(0, 1): {'left': array([0.]), 'right': array([1.]), 'node': array([4])},
+        (0, 2): {'left':array([0.5, 0.0]), 'right':array([1.0, 0.5]),
+        'node':array([3, 5])}}
+        ```
```

Comment:
In the output this line is included without a linebreak, so is very long - using a code block seems to work better:
```suggestion

        .. code-block:: python

            {
                (0, 1): {
                    "left": array([0.0]),
                    "right": array([1.0]),
                    "node": array([4]),
                },
                (0, 2): {
                    "left": array([0.5, 0.0]),
                    "right": array([1.0, 0.5]),
                    "node": array([3, 5]),
                },
            }

```

Reply:
thanks @benjeffery, this solves all of the above problems in one!